api-reference/10 UI Widgets/dxScheduler/1 Configuration/cellDuration.md

```diff
@@ -5,6 +5,6 @@ default: 30
 ---
 ---
 ##### shortDescription
-Specifies cell duration in minutes.
+Specifies cell duration in minutes. The interval between [startDayHour]({basewidgetpath}/Configuration/#startDayHour) and [endDayHour]({basewidgetpath}/Configuration/#endDayHour) should be divisible by this value.
```

Comment:
Hello. Please do not use {basewidgetpath} for links in doc members that belong to a certain UI component. In this case, replace them with links that lead to the Scheduler startDayHour and endDayHour properties.

Reply:
done